lib/config.go

```diff
@@ -57,6 +57,24 @@ const (
 	DefaultLogSizeMax = -1
 )
 
+// DefaultCapabilities for the capabilities option in the crio.conf file
+var DefaultCapabilities = []string{
+	"CHOWN",
+	"DAC_OVERRIDE",
+	"FSETID",
+	"FOWNER",
+	"MKNOD",
+	"NET_RAW",
+	"SETGID",
+	"SETUID",
+	"SETFCAP",
```

Comment:
I propose we remove SETFCAP from defaults, this is only used for installation software, like rpm.  The ability to create a setuid or file cap file should be blocked by default.